api/db/setup_local_vars.sh

```diff
@@ -1,4 +1,4 @@
 source setup_vars.sh
 export WORKBENCH_PASSWORD=${WORKBENCH_PASSWORD:=workbench!pwd}
-export DB_SERVER=${DB_SERVER:=localhost:3306}
```

Comment:
I agree. This is brittle right now because these values have to exactly match those used to start the database and yet they are defined in two separate places. To keep this PR small I might leave that to a follow-up PR, but if so it will be right on this one's heels.